Themes/default/Profile.template.php

```diff
@@ -125,6 +125,8 @@ function(data) {
 						ajax_indicator(false);
 						$("#alerts_menu_top span.amt").remove();
 						$("#alerts_menu div.alerts_unread").html(data);
+						if (typeof localStorage != "undefined")
+							localStorage.setItem("alertsCounter", 0);
```

Comment:
What will happen if the browser doesn't support localStorage?

Reply:
If the browser doesn't support localStorage then there'll be no need to update the localStorage's alertsCounter value in the first place, because there's be none

https://github.com/SychO9/SMF2.1/blob/ebbdad9f3806b1dc86fd912f9006dc0d5ee26478/Themes/default/scripts/alerts.js#L54-L55